README.md

```diff
@@ -5,11 +5,11 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/slok/reload)](https://goreportcard.com/report/github.com/slok/reload)
 [![Apache 2 licensed](https://img.shields.io/badge/license-Apache2-blue.svg)](https://raw.githubusercontent.com/slok/reload/master/LICENSE)
 
-`reload` is a universal mechanism to reload components in a Go application. Inspired by [oklog/run] and its simplicity, `reload` is a small Go library that has a simple API where `Notifiers` will trigger the reload process on the `Realoders`.
+`reload` is a universal mechanism to reload components in a Go application. Inspired by [oklog/run] and its simplicity, `reload` is a small Go library that has a simple API where `Notifiers` will trigger the reload process on the `Reloaders`.
 
-The mechanism is based on a reload manager that can have multiple notifiers, these notifiers will be executed and the manager will wait until one of those ends the execution. In that moment it will start the reload process. On the other hand the reload Manager can have multiple reloaders, that will be triggered when the reload process starts. This process will be runing in this manner forever until the reload Manager execution is stopped by an error or endidn the context passed on run.
+The mechanism is based on a reload manager that can have multiple notifiers. These notifiers will be executed and the manager will wait until one of those ends the execution. In that moment it will start the reload process. On the other hand the reload manager can have multiple reloaders, that will be triggered when the reload process starts. This process will be running in this manner forever until the reload manager execution is stopped by an error or end in the context passed on run.
```

Comment:
```suggestion
The mechanism is based on a reload manager that can have multiple notifiers. These notifiers will be executed and the manager will wait until one of those ends the execution. At that moment it will start the reload process. On the other hand, the reload manager can have multiple reloaders, that will be triggered when the reload process starts. This process will be running in this manner forever until the reload manager execution is stopped by an error or ending the context passed on run.
```